crates/astria-conductor/src/types.rs

```diff
@@ -34,7 +34,10 @@ impl SequencerBlockSubset {
             ..
         } = data.into_raw();
 
-        let our_rollup_data = rollup_data.remove(&namespace)?;
+        let Some(our_rollup_data) = rollup_data.remove(&namespace) else {
```

Comment:
Using `?` as it was before is idiomatic. No need to use the `let...else` construct.